src/client/datascience/notebook/helpers/executionHelpers.ts

```diff
@@ -48,8 +48,9 @@ export async function updateCellCode(cell: NotebookCell, text: string) {
 export async function addNewCellAfter(cell: NotebookCell, text: string) {
     await chainWithPendingUpdates(cell.notebook, (edit) => {
         traceCellMessage(cell, 'Create new cell after current');
-        edit.replaceNotebookCells(cell.notebook.uri, new NotebookRange(cell.index + 1, cell.index + 1), [
-            new NotebookCellData(NotebookCellKind.Code, text, cell.document.languageId, [], cell.metadata || {})
-        ]);
+        const cellData = new NotebookCellData(NotebookCellKind.Code, text, cell.document.languageId);
+        cellData.outputs = [];
+        cellData.metadata = cell.metadata || {};
```

Comment:
```suggestion
        cellData.metadata = {};
```

Reply:
@DonJayamanne Ahh, good catch. This was the same as the old code, but the old code looks like it has a bug. Don't want metadata from the old cell in the new one.